docs/root/intro/arch_overview/upstream/load_balancing/slow_start.rst

```diff
@@ -0,0 +1,60 @@
+.. _arch_overview_load_balancing_slow_start:
+
+Slow start mode
+===============
+
+Slow start mode is a configuration setting in Envoy to progressively increase amount of traffic for newly added upstream endpoints.
+With no slow start enabled Envoy would send a proportional amount of traffic to new upstream ednpoints.
```

Comment:
Can you fix the spelling in this doc as well, please. I'm not sure why the build didn't catch this.. I thought it used to.

Reply:
was surprising to me as well, perhaps spell check does not run on new .rst files

Reply:
I was also surprised that spell check did not catch those, perhaps it does not cover new .rst files...Fixed (with online spell checker)